lib/core/services/signets_api.dart

```diff
@@ -256,15 +257,23 @@ class SignetsApi {
         .findAllElements(_operationResponseTag(operation))
         .first;
 
-    // Throw exception if the error tag is not empty
+    // Throw exception if the error tag contains a blocking error
     if (responseBody
         .findElements(_signetsErrorTag)
         .first
         .innerText
         .isNotEmpty) {
-      throw ApiException(
-          prefix: tagError,
-          message: responseBody.findElements(_signetsErrorTag).first.innerText);
+      switch (responseBody.findElements(_signetsErrorTag).first.innerText) {
+        case SignetsError.scheduleNotAvailable:
```

Comment:
Il semble rentrer dans le "default" et non dans le case SignetsError.sceduleNotAvailable.

J'ai l'impression que le texte dans _responseBody.findElements(_signetsErrorTag).first.innerText_ n'est pas le même que dans 'SignetsError.scheduleNotAvailable'. Quand je change la constante scheduleNotAvailable il semble bien fonctionner :) 

J'ai print ce qu'il y a dans les deux textes et il y a des petites différence :) 
![image](https://user-images.githubusercontent.com/37625944/110471405-17619980-80aa-11eb-88aa-6e2f6f31a098.png)

Reply:
And this is why we would like an error code and no very long string XD

Reply:
I suggest two solutions:
1. We reverse the problem and instead of considering some error non-blocking (scheduleNotAvailable) and everything else blocking we can consider some error blocking (credentialsInvalid) and everything else non-blocking.
2. We can add the female version of the error in SignetsError and add it into the switch case.

I prefer solution 2. personally because we have more control over the errors but what do you think?

Reply:
Yes je suis d'accord avec l'option 2 :D

Reply:
Solution 2 for me also